ConfigurationSystem/Agent/VOMS2CSAgent.py

```diff
@@ -18,11 +16,29 @@
                                                               getAllUsers
 from DIRAC.Core.Utilities.Proxy                        import executeWithUserProxy
 
+__RCSID__ = "$Id$"
+
 class VOMS2CSAgent( AgentModule ):
 
-  def initialize( self ):
+  def __init__( self, *args, **kwargs ):
+    """ Defines default parameters
+    """
 
     self.__voDict = {}
+    self.__adminMsgs = {}
+    self.csapi = None
+    self.voChanged = False
+    self.dryRun = True
```

Comment:
So, basically you would swap the flags I have put here. I would be more in favor of having a default that is "same" everywhere, and at the same time make very clear points in the release notes for such a change.